setup.py

```diff
@@ -0,0 +1,35 @@
+from setuptools import setup
+
+
+with open('README.md') as file:
+    long_description = file.read()
+
+
+setup(
+    name="guacamole-files",
```

Comment:
The package name `guacamole` was already taken on pypi, I choose `guacamole-files` but it can be changed.